docs/tut-making-jettycat.md

```diff
@@ -137,7 +144,7 @@ For seasoned developers, actions might look as an extraneous step here, but they
 You can [read more about actions here](actions.html).
 :::
 
-Add this code to cat's On Step event:
+Add this code to the cat's Frame start event:
```

Comment:
This piece of code can be replaced with Action Down event, removing the  `if` clause.